spec/features/guests/show_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'rails_helper'
+
+RSpec.describe 'Guest Show' do 
+  describe 'as a visitor' do 
+    describe 'when I visit a guest show page' do 
+      it 'will have the guest name and a list of all rooms they have stayed in' do 
```

Comment:
You have great object setup in each test; you may want to consider using a `before :each` block with instance variables to cut down on the amount of typing you might be doing; not required just an idea!